lib/sycamore/sycamore/data/element.py

```diff
@@ -22,6 +22,16 @@ def __init__(self, element=None, /, **kwargs):
         if "properties" not in self.data:
             self.data["properties"] = {}
 
+    @property
+    def id(self) -> Optional[int]:
+        """A unique identifier for the element within a Document. Represents an order within the document"""
+        return self.data.get("id")
+
+    @id.setter
+    def id(self, value: int) -> None:
+        """Set the unique identifier of the element within a Document."""
+        self.data["id"] = value
```

Comment:
where is this actually set? also if it's just for ordering I might call it something like "seq_no"
might need to get plumbed through explode as well.

Reply:
I'll be setting it in the following PR in the partitioners which create Elements. Can rename to seq_no